### DISCUSSION TITLE
Meet & Greet

### DISCUSSION THREAD

**Discussion by syvwlch** [syvwlch/Data-Ignota#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cdrigby** [comment]
David Rigby here. I've worked in various areas of IT most of my life. My development experience is minimal. I have used GitHub to find & follow software projects that pique my interest, mostly related to the Linux operating system. 

I've known Syvwlch for almost 20 years and I am happy to kibbitz and discuss his project analyzing the text of Ada Palmer's novels, which I adore.

**Reply by syvwlch** [reply]
Hi David!

Folks, David is one of the long suffering souls who has been on the receiving end of long messaging rants when I'm nattering on about the books, or lately about how to apply TEI xml and Xquery and R to their analysis. Hopefully moving that gambit here will provide some relief. :-)

**Reply by cdrigby** [reply]
In life as in programming, [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns) is (a) good development! Our fellow bibliophiles will appreciate it.
